### DISCUSSION TITLE
🤔 how to add keys?

### DISCUSSION THREAD

**Discussion by gmorb** [madMAx43v3r/chia-plotter#131] [Q&A]
I keep getting needs to be specified via -f/-p hex. When i input my keys without the <> it loops back to the same message when adding with <> i get a syntax error. 😩

Noob here sorry. Also this see chia keys show command i have no idea how to get it to run either. Any help appreciated :)

**Comment by madMAx43v3r** [comment]
See https://github.com/madMAx43v3r/chia-plotter/issues/132

**Comment by gmorb** [comment]
So i installed the chia blockchain gui not sure if it was necessary cause i already had my keys. Then i figured out i had to input all the commands together. I was placing my commands one by one which is why i kept getting errors. For anyone as noob as me hope this helps.

Example: ./build/chia_plot -p (pool key here) -f (farm key here) t- (temp directory here where your temp plots will be written) -d (final destination for completed plots this would be an external drive or wherever you plan to keep your plots for farming) -n (number of plots here i used 8) -r (number of threads here i used 32 depends on your cpu capabilities)
